.azure/pipelines/helix-matrix.yml

```diff
@@ -0,0 +1,63 @@
+# We only want to run full helix matrix on master
+trigger:
+  batch: true
+  branches:
+    include:
+    - master
+    - release/*
+
+# Run PR validation for now
+pr:
+  autoCancel: true
+  branches:
+    include:
+    - '*'
+    
+variables:
+  - name: _UseHelixOpenQueues
+    value: 'true'
```

Comment:
Suggest handling internal builds in this YAML e.g. making this value conditional and including `- group: DotNet-HelixApi-Access`. It's not required at this point (unless others disagree) but would make it easier to add an internal pipeline in the future.

Reply:
Well, they were there before, i removed them because you asked why they were there :)  But sure I can put it back

Reply:
My apologies, I wasn't clear in that comment. I was asking whether we intended to set up an internal pipeline, not to remove the code supporting one.

Reply:
I don't see a big need for an internal pipeline right now.